examples/django/posts/urls.py

```diff
@@ -1,10 +1,11 @@
-from django.conf.urls import url, include
+from django.conf.urls import include
+from django.urls import re_path
 
 from .api import PostResource
 
 
 urlpatterns = [
-    url(r'^posts/', include(PostResource.urls())),
+    re_path(r'^posts/', include(PostResource.urls())),
```

Comment:
So... you found another place that [this PR](https://github.com/toastdriven/restless/pull/134/files) missed it! 😅

Reply:
Ah... this is the examples folder 😛